<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cAxisPage</class>
 <widget class="QWidget" name="cAxisPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>监控</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>调试</string>
      </attribute>
      <widget class="jmcSetPosWidget" name="jmcsetpos" native="true">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>380</y>
         <width>350</width>
         <height>150</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#jmcsetpos{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcResetWidget" name="jmcreset" native="true">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>450</y>
         <width>350</width>
         <height>120</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#jmcreset{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcJogWidget" name="jog" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>350</width>
         <height>270</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#jog{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcMoveVelWidget" name="movevel" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>450</y>
         <width>350</width>
         <height>240</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#movevel{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcMoveAbsWdiget" name="moveabs" native="true">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>100</y>
         <width>350</width>
         <height>270</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#moveabs{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcWtmtCCWidget" name="jmccc" native="true">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>540</y>
         <width>350</width>
         <height>150</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#jmccc{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcHomeWidget" name="jmchome" native="true">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>290</y>
         <width>350</width>
         <height>150</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#jmchome{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcStopWidget" name="jmcstop" native="true">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>100</y>
         <width>350</width>
         <height>180</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#jmcstop{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="jmcPowerWidget" name="power" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>350</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#power{
    background-color: #888888; /* 更浅的灰色背景 */
    color: #ffffff; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border:2px solid #2c2c2c;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
       </property>
       <property name="text">
        <string>实际位置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="cSuperBtn" name="fPos">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>40</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
       </property>
       <property name="text">
        <string>实际速度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="cSuperBtn" name="fVel">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>40</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_9">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>40</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
       </property>
       <property name="text">
        <string>实际加速度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="cSuperBtn" name="fAcc">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>40</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_10">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
       </property>
       <property name="text">
        <string>实际扭矩</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="cSuperBtn" name="sTorque">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_11">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>70</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
       </property>
       <property name="text">
        <string>控制模式</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="cSuperBtn" name="mode">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>70</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_12">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>70</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;

</string>
       </property>
       <property name="text">
        <string>状态机</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="cSuperBtn" name="state">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>70</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> 
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
     border: 2px solid #b0b0b0;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cSuperBtn</class>
   <extends>QPushButton</extends>
   <header>include/cwidget/cSuperBtn.h</header>
  </customwidget>
  <customwidget>
   <class>jmcPowerWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcpowerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcJogWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcjogwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcMoveAbsWdiget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcmoveabswdiget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcMoveVelWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcmovevelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcSetPosWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcsetposwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcWtmtCCWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcwtmtccwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcStopWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcstopwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcHomeWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmchomewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>jmcResetWidget</class>
   <extends>QWidget</extends>
   <header>include/csoftmotion/jmcresetwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
